features/flows/rest/impl/src/main/java/org/opennms/netmgt/flows/rest/internal/FlowRestServiceImpl.java

```diff
@@ -327,4 +329,26 @@ private static void populateResponseFromTable(Table<?, Long, Double> table, Flow
         response.setValues(values);
     }
 
+    @Override
+    public String getDeepDiveToolUrl(UriInfo uriInfo) {
+
+        long count = waitForFuture(
+                flowRepository.getFlowCount(getFiltersFromQueryString(uriInfo.getQueryParameters())));
+
+        if (count > 0) {
+            // If there are valid flows then generate URL from template.
+            String deepDiveToolUrl = System.getProperty("org.opennms.netmgt.flows.deepDiveUrl");
```

Comment:
Can we set the property when instantiating the repository. 
We should probably provide some defaults, as otherwise the deep dive url would not work by default, but OpenNMS usually provides some behaviour if a certain feature is enabled.